### PR TITLE
Driver ed landing

### PR DESCRIPTION
**Pull request by adamrmelnyk**
* Changes for the new Drivers ed Landing page

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:book:

File: app/helpers/ama_layout_path_helper.rb [hunk 1]
```diff
@@ -64,6 +64,6 @@ def membership_esso_reload_path
   end
 
   def driveredonline_path
-    "#{Rails.configuration.driveredonline_site}/login"
+    "#{Rails.configuration.driveredonline_site}"
```
Comment (MathieuGilbert) [inline app/helpers/ama_layout_path_helper.rb]: Can drop the `"#{ }"` parts to just return that value directly, since there's no more plain text.

**Comment by MathieuGilbert** [comment]
:hankey:

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Comment by adamrmelnyk** [comment]
Missed a gem version bump :wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**